pkg/cmd/step/helm/step_helm_apply.go

```diff
@@ -313,6 +313,39 @@ func (o *StepHelmApplyOptions) Run() error {
 		return err
 	}
 
+	// Now let's unpack all the dependencies and apply the vault URLs
+	dependencies, err := filepath.Glob(filepath.Join(dir, "charts", "*.tgz"))
```

Comment:
Does this fail when `charts` folder doesn't exit? e.g. for a chart without dependencies

Reply:
No. You can see the docs for `filepath.Glob` at https://golang.org/pkg/path/filepath/#Glob where it says that it will return `nil` when there are no matches. Loops in Go can cope with nil slices.